app/pb_kits/playbook/pb_progress_simple/progress_simple.rb

```diff
@@ -2,99 +2,97 @@
 
 module Playbook
   module PbProgressSimple
-    class ProgressSimple < Playbook::PbKit::Base
-      PROPS = %i[configured_align
-                 configured_classname
-                 configured_data
-                 configured_id
-                 configured_max
-                 configured_muted
-                 configured_percent
-                 configured_value
-                 configured_width].freeze
-
-      def initialize(align: default_configuration,
-                     classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     max: default_configuration,
-                     muted: default_configuration,
-                     percent: default_configuration,
-                     value: default_configuration,
-                     width: default_configuration)
-        self.configured_align = align
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_max = max
-        self.configured_muted = muted
-        self.configured_percent = percent
-        self.configured_value = value
-        self.configured_width = width
-      end
+    class ProgressSimple
+      include Playbook::Props
 
-      def width
-        default_value(configured_width, "100%")
-      end
+      class ProgressError < StandardError; end
+
+      partial "pb_progress_simple/progress_simple"
+
+      prop :align, type: Playbook::Props::Enum,
+           values: %w[left center right],
+           default: "left"
+      prop :value, type: Playbook::Props::Number
+      prop :max, type: Playbook::Props::Number
+      prop :muted, type: Playbook::Props::Boolean,
+           default: false
+      prop :percent, type: Playbook::Props::Percentage
+      # :width prop should not probably be a string type
+      # Should we be allowing the user to pass this value at all?
+      # could this possibly be [sm, md, lg]?
+      prop :width, default: "100%"
```

Comment:
Width should be an integer from 0-100 if the plan is for it to be this configurable.. but my question is why do we even need this prop at all? Shouldn't `percent` prop take care of this?

Reply:
The `width` prop is used here - https://github.com/powerhome/playbook/pull/364/files#diff-546b67958995ee7824144b42d48a0e79R44 - and seems to provide stylistic configuration for the size of the full progress bar container. `percent` OR the combo of `max` & `value` sets how much actual progress (blue) is in the bar.

We talked about this prop A LOT, and finally decided to leave a note instead of make an architectural change. The reason for that is we don't yet have any use cases in Nitro. One though of mine is that the developer using this kit could change the "width" of how long the progress bar is by simply putting in any sort of element with a fixed width. - Which makes my vote to deal with this lean towards removing this prop entirely.

Thoughts are welcome on this, but we don't want to get to bogged down. If we can come to a consensus, we are happy to open up another PR and address `width` specifically.